05/src/components/Nav.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Menu from './Menu';
+
+
+
+class Nav extends React.Component {
+    render(){
+        return <nav> <Menu /> </nav>

```

Comment:
Tutaj wywołanie powinno wyglądać w ten sposób:
```
<nav>
    <Menu 
        items={[
            {text: 'strona główna', url: '/'}, 
            {text: 'www', url: '/www'}
        ]}
    />
</nav>
```
Czyli w komponencie `<Menu>` będą dostępne dane pod lokalizacją `this.props.items`.